### ISSUE TITLE
Audio player restart from begining after pause and Android sleep start

### ISSUE THREAD

**Issue by Spheerys** [nextcloud/android#11809]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

1. load a mp3 file on Nextcloud for Android and launch it
2. the sound is working well
3. press pause
4. wait until the android sleeping mode start (black screen generally)
5. awake the phone

### Expected behaviour

The player should be paused at the same point as step 3 and you should be able to play again at the paused point

### Actual behaviour

The player is restarting the audio file from the beginning

### Android version

12

### Device brand and model

Fairphone 3

### Stock or custom OS?

Custom (explain in "additional information")

### Nextcloud android app version

eOS v 1.12

### Nextcloud server version

3.25.0

### Using a reverse proxy?

I don't know

#### Cause of error
```java
Exception in thread "main" java.lang.NullPointerException: Attempt to invoke interface method 'boolean com.owncloud.android.ui.adapter.CommonOCFileListAdapterInterface.isMultiSelect()' on a null object reference
    at com.owncloud.android.ui.fragment.OCFileListFragment.onItemClicked(OCFileListFragment.java:972)
    at com.owncloud.android.ui.activity.FileDisplayActivity.onOpenFileIntent(FileDisplayActivity.java:586)
    at com.owncloud.android.ui.activity.FileDisplayActivity.onNewIntent(FileDisplayActivity.java:525)
    at android.app.Activity.performNewIntent(Activity.java:8073)
    at android.app.Instrumentation.callActivityOnNewIntent(Instrumentation.java:1445)
    at android.app.Instrumentation.callActivityOnNewIntent(Instrumentation.java:1458)
    at android.app.ActivityThread.deliverNewIntents(ActivityThread.java:3904)
    at android.app.ActivityThread.handleNewIntent(ActivityThread.java:3911)
    at android.app.servertransaction.NewIntentItem.execute(NewIntentItem.java:56)
    at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
    at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
    at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2253)
    at android.os.Handler.dispatchMessage(Handler.java:106)
    at android.os.Looper.loopOnce(Looper.java:201)
    at android.os.Looper.loop(Looper.java:288)
    at android.app.ActivityThread.main(ActivityThread.java:7870)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)

```

#### App information
* ID: `com.nextcloud.client`
* Version: `30250090`
* Build flavor: `generic`

#### Device information
* Brand: `Fairphone`
* Device: `FP3`
* Model: `FP3`
* Id: `SQ3A.220705.004`
* Product: `lineage_FP3`

#### Firmware
* SDK: `32`
* Release: `12`
* Incremental: `eng.root.20230615.041651`

**Comment by joshtrichards** [comment]
Cc: @alperozturk96 This one and #12391 look similar and are very close to a different NPE you recently fixed in #13208.

**Comment by alperozturk96** [comment]
> Cc: @alperozturk96 This one and #12391 look similar and are very close to a different NPE you recently fixed in #13208.

Seems like different problem.
